Always use apt full-upgrade and simplify the NVMe updates (#3962)
raspberrypi/documentation: documentation/asciidoc/computers/raspberry-pi/boot-eeprom.adoc

Advanced users can switch to the `latest` bootloader to get the latest functionality.

First, ensure that your Raspberry Pi runs the latest software. Run the following command to update:

[source,console]
----
$ sudo apt update && sudo apt full-upgrade
----

> Run the following command to start `raspi-config`.

Reviewer: ```suggestion
Next, run the following command to open `raspi-config`:
```

[source,console]
----
$ sudo raspi-config
----

> Navigate to `Advanced Options` and then `Bootloader Version`. Select `Latest` and choose `Yes` to confirm. Select `Finish` and confirm you want to reboot.

Reviewer: ```suggestion
Navigate to `Advanced Options` > `Bootloader Version`. Select `Latest`, then choose `Yes` to confirm. Select `Finish` and confirm that you want to reboot.
```

| Bootloader EEPROM image

| `pieeprom.bin`

> | Bootloader EEPROM image - same as pieeprom.upd but changes recovery.bin behaviour to not rename itself to `RECOVERY.000`.

Reviewer: ```suggestion
| Bootloader EEPROM image - same as `pieeprom.upd` but changes `recovery.bin` behaviour to not rename itself to `RECOVERY.000`.
```
